docs/effects/lighting-effect.md

```diff
@@ -0,0 +1,102 @@
+# LightingEffect
+
+The Lighting Effect applies ambient, point and directional lighting to layers which support material property.
+
+```js
+import DeckGL, {GeoJsonLayer, LightingEffect} from '@deck.gl/core';
+import {AmbientLight, PointLight, DirectionalLight, PhongMaterial} from 'luma.gl';
+
+// create ambient light source
+const ambientLight = new AmbientLight({
+  color: [255, 255, 255],
+  intensity: 1.0});
+// create point light source
+const pointLight = new PointLight({
+  color: [255, 255, 255],
+  intensity: 2.0,
+  // use coordinate system as the same as view state
+  position: [-125, 50.5, 5000]
+});
+// create directional light source
+const directionalLight = new DirectionalLight({
+  color: [255, 255, 255],
+  intensity: 1.0,
+  direction: [-3, -9, -1]
+});
+
+const lightingEffect = new LightingEffect({ambientLight, pointLight, directionalLight});
```

Comment:
`pointLights` and `directionalLights` are supposed to be arrays?

Reply:
no, use Object like {ambientLight, pointlight1, pointlight2, directionalLight1, directionlLight2} is sufficient.

Reply:
That does not match your properties documentation in this file.

Reply:
added a section for constructor